app/src/main/java/appricottsoftware/clarity/sync/ClarityApp.java

```diff
@@ -21,7 +21,7 @@ public void onCreate() {
     // Get the single instance of the API client
     public static ClarityClient getRestClient() {
         if(clarityClient == null) {
-            clarityClient = new ClarityClient();
+            clarityClient = new ClarityClient(context);
```

Comment:
Why is there a context parameter for the ClarityClient constructor if it's not used in ClarityClient?

Reply:
I believe we needed that in order to use the getString(R.string.stringname) from the key.xml?